jx_sqlite/snowflake.py

```diff
@@ -113,19 +113,25 @@ def change_schema(self, required_changes):
         :return: None
         """
         required_changes = wrap(required_changes)
+        c=[]
         for required_change in required_changes:
             if required_change.add:
-                self._add_column(required_change.add)
+                command = self._add_column(required_change.add)
+                if command!=None:
+                    c.append(command)
             elif required_change.nest:
```

Comment:
Oh!  I had forgotten there was a (not working) `{"nest": params}` command specifically for moving columns from he fact table out to the arms of the snowflake